lib/msal-browser/docs/testing.md

```diff
@@ -0,0 +1,143 @@
+# Testing
+
+MSAL Browser starting version 2.17.0 has added the `loadTokens()` API, which allows the loading of id tokens and access tokens to the MSAL cache, which can then be read out using `acquireTokenSilent()`. 
+
+**Note: This is an advanced feature that is intended for testing purposes only. Loading tokens to your application's cache may cause your app to break.**
+
+## The loadTokens() API
+
+The `loadToken()` API can be accessed by calling `getTokenCache()` on MSAL Browser's `PublicClientApplication` instance. 
+
+```js
+myMSALObj.getTokenCache().loadTokens(silentRequest, serverResponse, loadTokenOptions);
```

Comment:
Is it better to make it:

```suggestion
const msalTokenCache = myMSALObj.getTokenCache();
msalTokenCache.loadTokens(silentRequest, serverResponse, loadTokenOptions);
```

Reply:
Essentially the whole sample can now use the `msalTokenCache` to make other calls to fetch data:

```javascript
msalTokenCache.getAllAccounts()
```

Reply:
Good point. I changed the doc to match. I felt this PR was getting to big, so I will move `getAllAccounts()` in another PR, but yes, eventually they will use the tokenCache to make other calls to fetch data.